tool-metagen/src/DA/Daml/GenDaml.hs

```diff
@@ -49,8 +49,14 @@ ppDecl (EnumType name constrs comment) =
 ppDecl (RecordType name fields comment) =
     ppComment Before comment
     $$ text "data" <+> text name <+> text "=" <+> text name <+> text "with"
-        $$ nest 4 (ppRecordFields fields
-                  $$ nest 2 (text "deriving (Eq, Ord, Show)"))
+        $$ nest 4 (ppRecordFields fields)
+    $$ text ""
+    $$ text "instance Eq" <+> text name <+> text "where" <+> text "(==) = GHC.Types.primitive @\"BEEqual\""
```

Comment:
Are you sure that this will work for derived types?

What was the reason for this change in the first place? Is it just to make `show` more legible?